content/doc/book/pipeline/shared-libraries.adoc

```diff
@@ -99,22 +99,26 @@ image::pipeline/add-global-pipeline-libraries.png[alt = "Jenkins System Configur
 Since these libraries will be globally usable, any Pipeline in the system can
 utilize functionality implemented in these libraries.
 
-These libraries are considered "trusted:" they can run any methods in Java,
+These libraries may be marked “trusted“, meaning they can run any methods in Java,
 Groovy, Jenkins internal APIs, Jenkins plugins, or third-party libraries.  This
 allows you to define libraries which encapsulate individually unsafe APIs in a
 higher-level wrapper safe for use from any Pipeline.  Beware that **anyone able to
 push commits to this SCM repository could obtain unlimited access to Jenkins**.
 You need the _Overall/RunScripts_ permission to configure these libraries
 (normally this will be granted to Jenkins administrators).
```

Comment:
```suggestion
These libraries may be marked “trusted“, meaning they can run any methods in Java,
Groovy, Jenkins internal APIs, Jenkins plugins, or third-party libraries.
This allows you to define libraries that encapsulate individually unsafe APIs in a
higher-level wrapper that is safe for use from any pipeline.
Beware that **anyone able to push commits to this SCM repository could obtain unlimited access to Jenkins**.
You need the _Overall/RunScripts_ permission to configure these libraries (normally this will be granted to Jenkins administrators).
```